lib/extras/dec/pnm.cc

```diff
@@ -323,8 +312,77 @@ Span<const uint8_t> MakeSpan(const char* str) {
   return Bytes(reinterpret_cast<const uint8_t*>(str), strlen(str));
 }
 
+void ReadLinePNM(void* opaque, size_t xpos, size_t ypos, size_t xsize,
```

Comment:
I am not sure I like the fact that the size of the buffer is inferred from xsize + opaque... What do you think about passing in the buffer size, and assert-failing if it does not match?

Reply:
Done.